### DISCUSSION TITLE
burg/burg themes in ventoy

### DISCUSSION THREAD

**Discussion by rvvh472** [ventoy/Ventoy#1360] [Ideas]
Hi
I love the way ventoy works. But To me grub menu seems old. I would love to have the burg (apple like) boot menu style. What if we install burg & burg themes in Ventoy. Three ways for boot menu
1) List View (Already has)
2) Tree View (Already has)
3) Burg View (suggested)
_____ with feature at least:  default boot entries e.g. If someone only want to show 2 VHD(x) at boot up and press space to shows all boot entries in burg layout

**Comment by warpedgeoid** [comment]
I'd like to see this too. I'm planning to use Ventoy for an image deployment flash drive that will be used by others within my organization, and I think they'd find the burg menu a bit easier to understand.
